core/ecschema-editing/src/Editing/Phenomenons.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+import { PhenomenonProps,SchemaKey, SchemaMatchType } from "@bentley/ecschema-metadata";
+import { SchemaContextEditor, SchemaItemEditResults } from "./Editor";
+import { MutablePhenomenon } from "./Mutable/MutablePhenomenon";
+import { MutableSchema } from "./Mutable/MutableSchema";
+
+/**
+ * @alpha
+ * A class allowing you to create schema items of type Phenomenon.
+ */
+export class Phenomenons {
```

Comment:
The plural of `phenomenon` is `phenomena`.

Reply:
I was just moving the code, but that is painful to look at.  I will update.

Reply:
done